configuration/conf/config.properties

```diff
@@ -4,7 +4,7 @@
 
 ################### 被测数据库配置 #######################
 # 被测试的数据库，目前的格式为{name}{-version}{-insert mode}(注意-号)其全部参考值参见README文件，注意版本需要匹配
-DB_SWITCH=IoTDB-110-SESSION_BY_TABLET
+DB_SWITCH=IoTDB-013-SESSION_BY_RECORDS
```

Comment:
There are no need to update default config

Reply:
sure